content/content-strategy/set-up-governance/accountability-ownership/new-long-description.md

```diff
@@ -2,7 +2,7 @@
 layout: blocks/long_description
 title: Content teams - accountability, expertise, escalation
 ---
-#### Long text description
+#### [1]Long text description
```

Comment:
Looks like this file still has 'Long text description' in it. @sirleech

Reply:
Roger. Will fix